### PR TITLE
AVRO-3546: Add favicons to the website

### PR DESCRIPTION
**Pull request by RyanSkraba**
Adds the favicons to the website

### Jira

- [X] My PR addresses AVRO-3546

### CODE REVIEW

File: doc/layouts/partials/favicons.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+-->
+<link rel="apple-touch-icon" sizes="57x57" href="/https://apache.org/favicons/apple-touch-icon-57x57.png">
```
Comment (martin-g) [inline doc/layouts/partials/favicons.html:22]:
```suggestion
<link rel="apple-touch-icon" sizes="57x57" href="https://apache.org/favicons/apple-touch-icon-57x57.png">
```

File: doc/layouts/partials/favicons.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<link rel="apple-touch-icon" sizes="60x60" href="/https://apache.org/favicons/apple-touch-icon-60x60.png">
```
Comment (martin-g) [inline doc/layouts/partials/favicons.html:23]:
```suggestion
<link rel="apple-touch-icon" sizes="60x60" href="https://apache.org/favicons/apple-touch-icon-60x60.png">
```

File: doc/layouts/partials/favicons.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+<link rel="apple-touch-icon" sizes="72x72" href="/https://apache.org/favicons/apple-touch-icon-72x72.png">
```
Comment (martin-g) [inline doc/layouts/partials/favicons.html:24]:
```suggestion
<link rel="apple-touch-icon" sizes="72x72" href="https://apache.org/favicons/apple-touch-icon-72x72.png">
```

File: doc/layouts/partials/favicons.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+<link rel="apple-touch-icon" sizes="76x76" href="/https://apache.org/favicons/apple-touch-icon-76x76.png">
```
Comment (martin-g) [inline doc/layouts/partials/favicons.html:25]:
```suggestion
<link rel="apple-touch-icon" sizes="76x76" href="https://apache.org/favicons/apple-touch-icon-76x76.png">
```

File: doc/layouts/partials/favicons.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (RyanSkraba) [reply doc/layouts/partials/favicons.html]: No, it wasn't!  Good catch!

**Pull request merged**